### ISSUE TITLE
bug: Peeking into queues yields null ReceiveMessageResult

### ISSUE THREAD

**Issue by DAcodedBEAT** [localstack/localstack#8825]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I have LocalStack running in docker and I created a FIFO queue via `awslocal sqs create-queue`.  In my application, it sends messages to this queue and acts on it but when using the peek functionality (`/_aws/sqs/messages`), I receive the following output:
```json
{
  "ReceiveMessageResponse": {
    "ReceiveMessageResult": null,
    "ResponseMetadata": {
      "RequestId": "<generated-request-id>"
    }
  }
}
```

### Expected Behavior

`ReceiveMessageResult` should show the messages that are on the queue

### How are you starting LocalStack?

With a docker-compose file

#### How are you starting localstack (e.g., `bin/localstack` command, arguments, or `docker-compose.yml`)

    docker-compose up -d localstack

#### Client commands (e.g., AWS SDK code snippet, or sequence of "awslocal" commands)

    awslocal sqs create-queue --queue-name "local-queue.fifo"  --attributes FifoQueue=true,ContentBasedDeduplication=true,DelaySeconds=15,MessageRetentionPeriod=600,ReceiveMessageWaitTimeSeconds=5,VisibilityTimeout=900,DeduplicationScope=messageGroup,FifoThroughputLimit=perQueue
      
    curl -H "Accept: application/json" "http://localhost:4566/_aws/sqs/messages/us-east-1/000000000000/local-queue.fifo" | jq

### Environment

```markdown
- OS: macOS
- LocalStack: latest
```

### Anything else?

I am attempting to follow this documentation: https://docs.localstack.cloud/user-guide/aws/sqs/#peeking-into-queues

**Comment by DAcodedBEAT** [comment]
> We recommend joining our [Slack Community](https://localstack.cloud/contact/) for real-time help and drop a message to LocalStack Pro Support if you are a Pro user! 

I tried to join the slack community but it requires that I have a `@localstack.cloud` email address 😭

**Comment by DAcodedBEAT** [comment]
@localstack sorry for the org-tag, but it feels like I'm stuck here, can someone help me out in the process?

**Comment by thrau** [comment]
hey Aron @DAcodedBEAT, sorry for not getting back sooner! since your queue has `DelaySeconds=15`, messages will appear in the result only after 15 seconds. in october we merged #9357 which allows you specify the parameter `ShowDelayed`, which will also return the delayed messages. here's a [link to the docs](https://docs.localstack.cloud/user-guide/aws/sqs/#show-invisible-or-delayed-messages).

so the following will work (double checked with your example):
```console
awslocal sqs send-message --queue-url http://sqs.us-east-1.localhost.localstack.cloud:4566/000000000000/local-queue.fifo \
  --message-body 'hello' --message-group-id "1"
```
```console
curl -H "Accept: application/json" "http://localhost:4566/_aws/sqs/messages/us-east-1/000000000000/local-queue.fifo?ShowDelayed=true" | jq
```
```json
{
  "ReceiveMessageResponse": {
    "ReceiveMessageResult": {
      "Message": {
        "MessageId": "996f9583-eea3-461f-8f94-e22d29760e8a",
        "MD5OfBody": "5d41402abc4b2a76b9719d911017c592",
        "Body": "hello",
        "Attribute": [
          {
            "Name": "SenderId",
            "Value": "000000000000"
          },
          {
            "Name": "SentTimestamp",
            "Value": "1704215863017"
          },
          {
            "Name": "MessageGroupId",
            "Value": "1"
          },
          {
            "Name": "MessageDeduplicationId",
            "Value": "2cf24dba5fb0a30e26e83b2ac5b9e29e1b161e5c1fa7425e73043362938b9824"
          },
          {
            "Name": "SequenceNumber",
            "Value": "14639102793818832896"
          },
          {
            "Name": "ApproximateReceiveCount",
            "Value": "0"
          },
          {
            "Name": "ApproximateFirstReceiveTimestamp",
            "Value": "0"
          },
          {
            "Name": "IsDelayed",
            "Value": "true"
          }
        ],
        "ReceiptHandle": "SQS/BACKDOOR/ACCESS"
      }
    },
    "ResponseMetadata": {
      "RequestId": "c8406a9e-2fb8-405f-bc1e-27608e9ee9f7"
    }
  }
}
```

closing this issue as solved!
